frontend/src/ui/Breadcrumbs.tsx

```diff
@@ -13,20 +14,24 @@ type Segment = {
     href: string,
 }
 
-export const Breadcrumbs: React.FC<Props> = ({ path }) => (
-    <nav aria-label="breadcrumbs" css={{ marginBottom: 16 }}>
-        <ol css={{ listStyle: "none", padding: 0, margin: 0 }}>
-            <Segment target="/" first active={path.length == 0}>
-                <FontAwesomeIcon title="Home" icon={faHome} />
-            </Segment>
-            { path.map((segment, i) => (
-                <Segment key={i} target={segment.href} active={i == path.length - 1}>
-                    { segment.label }
+export const Breadcrumbs: React.FC<Props> = ({ path }) => {
+    const { t } = useTranslation();
+
+    return (
+        <nav aria-label="breadcrumbs" css={{ marginBottom: 16 }}>
+            <ol css={{ listStyle: "none", padding: 0, margin: 0 }}>
+                <Segment target="/" first active={path.length == 0}>
```

Comment:
```suggestion
                <Segment target="/" first active={path.length === 0}>
```

But this was there, before. How could that happen!? :scream:

Reply:
Mhhhhh eslint does not emit anything :/

Reply:
Yeah we need a proper config. >_<

I mean it's fine, right. With TypeScript we even know that `path.length` is a `number`, so nothing surprising should ever happen here (apart from the ever looming fact that TS's typesystem is unsound ...). But of course `===` is also faster, because there is no coercion involved. :wink: